### ISSUE TITLE
Cluster verify reports N+1 errors for offline nodes

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#194]
Originally reported of Google Code with ID 138.

```
On a cluster which is N+1 compliant, transitioning a node into offline state creates false N+1 errors:


node1# gnt-cluster verify
Tue Feb  8 06:54:46 2011 * Verifying global settings
Tue Feb  8 06:54:46 2011 * Gathering data (4 nodes)
Tue Feb  8 06:54:49 2011 * Gathering disk information (4 nodes)
Tue Feb  8 06:54:51 2011 * Verifying node status
Tue Feb  8 06:54:51 2011 * Verifying instance status
Tue Feb  8 06:54:51 2011 * Verifying orphan volumes
Tue Feb  8 06:54:51 2011 * Verifying orphan instances
Tue Feb  8 06:54:51 2011 * Verifying N+1 Memory redundancy
Tue Feb  8 06:54:51 2011 * Other Notes
Tue Feb  8 06:54:51 2011   - NOTICE: 6 non-redundant instance(s) found.
Tue Feb  8 06:54:51 2011 * Hooks Results
node1# gnt-node modify -O yes node3
Modified node node3
 - offline -> True
node1# gnt-cluster verify
Tue Feb  8 06:55:07 2011 * Verifying global settings
Tue Feb  8 06:55:07 2011 * Gathering data (4 nodes)
Tue Feb  8 06:55:10 2011 * Gathering disk information (4 nodes)
Tue Feb  8 06:55:11 2011 * Verifying node status
Tue Feb  8 06:55:11 2011 * Verifying instance status
Tue Feb  8 06:55:11 2011   - ERROR: instance instance14: couldn't retrieve status for disk/0 on node3: node offline
Tue Feb  8 06:55:11 2011   - ERROR: instance instance14: instance lives on offline node(s) node3
Tue Feb  8 06:55:11 2011   - ERROR: instance instance15: couldn't retrieve status for disk/0 on node3: node offline
Tue Feb  8 06:55:11 2011   - ERROR: instance instance15: instance lives on offline node(s) node3
Tue Feb  8 06:55:11 2011   - ERROR: instance instance16: instance lives on offline node(s) node3
Tue Feb  8 06:55:11 2011   - ERROR: instance instance17: instance lives on offline node(s) node3
Tue Feb  8 06:55:11 2011 * Verifying orphan volumes
Tue Feb  8 06:55:11 2011 * Verifying orphan instances
Tue Feb  8 06:55:11 2011 * Verifying N+1 Memory redundancy
Tue Feb  8 06:55:11 2011   - ERROR: node node3: not enough memory to accomodate instance failovers should node node1 fail
Tue Feb  8 06:55:11 2011   - ERROR: node node3: not enough memory to accomodate instance failovers should node node2 fail
Tue Feb  8 06:55:11 2011 * Other Notes
Tue Feb  8 06:55:11 2011   - NOTICE: 6 non-redundant instance(s) found.
Tue Feb  8 06:55:11 2011   - NOTICE: 1 offline node(s) found.
Tue Feb  8 06:55:12 2011 * Hooks Results

This should be fixed, as it is too verbose. Also, the extra disk errors are not helpful.
```
Originally added on 2011-02-08 14:56:15 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
This was fixed, and will be release in 2.4.
```
Originally added on 2011-02-09 11:24:36 +0000 UTC.

Changed State: Fixed
Added to Milestone: Release2.4

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:08 +0000 UTC.

Changed State: Released
